runtime/libffi/z/ffi.c

```diff
@@ -267,31 +267,36 @@ ffi_prep_args (unsigned char *stack, extended_cif *ecif)
 	    *(unsigned char *) arg_ptr = * (unsigned char *) (* p_argv);
 					arg_ptr += 3;
 	    break;
- 
+
 	  case FFI_TYPE_SINT8:
 	    *(signed char *) arg_ptr = * (signed char*) (* p_argv);
 			arg_ptr += 3;
 	    break;
- 
+
 	  default:
 	    FFI_ASSERT (0);
 	    break;
         }
       arg_ptr += size;
     }
-  
+
 }
 
 /*======================== End of Routine ============================*/
- 
+ffi_status
```

Comment:
In `ffi.c` file, this is the only change - I have not yet changed anything functionally for 31-bit z/OS (Given that we do not have support for struct parameters in 31-bit z/OS) - Though the variadic argument change for double needs to be changed here - I need to check with ABI document w.r.t 31-bit z/OS before making any functional change.

Rest of the changes in the files are removal of white space at the end of line.